src/image.c

```diff
@@ -657,6 +675,10 @@ int wolfBoot_verify_integrity(struct wolfBoot_image *img)
         return -1;
     if (image_hash(img, digest) != 0)
         return -1;
+#if defined(WOLFBOOT_TPM) && defined(WOLFBOOT_MEASURED_BOOT)
```

Comment:
Consider putting this after the memcmp, so its only updated on success.

Reply:
I had this comment in my initial code, but removed it in the last moment. Perhaps I should re-add it?
+    /*
+     * TPM Measurement must be performed regardless of the
+     * verification outcome afterwards, becuse the purpose
+     * of a Measured Boot is to record the current boot state
+     */

Reply:
Yes add this comment.